specs/latest/2.0/index.html

```diff
@@ -755,11 +755,12 @@ <h2 class="no-toc">Table of contents</h2>
   void uniformMatrix3x4fv(WebGLUniformLocation? location, GLboolean transpose, UniformMatrixFVSource value);
   void uniformMatrix4x3fv(WebGLUniformLocation? location, GLboolean transpose, UniformMatrixFVSource value);
   void vertexAttribI4i(GLuint index, GLint x, GLint y, GLint z, GLint w);
+  typedef (Int32Array or sequence&lt;GLint&gt;) VertexAttribIVSource;
   void vertexAttribI4iv(GLuint index, Int32Array v);
-  void vertexAttribI4iv(GLuint index, sequence&lt;GLint&gt; v);
+  void vertexAttribI4iv(GLuint index, VertexAttribIVSource value);
   void vertexAttribI4ui(GLuint index, GLuint x, GLuint y, GLuint z, GLuint w);
-  void vertexAttribI4uiv(GLuint index, Uint32Array);
-  void vertexAttribI4uiv(GLuint index, sequence&lt;GLuint&gt; v);
+  typedef (Uint32Array or sequence&lt;GLuint&gt;) VertexAttribUIVSource;
+  void vertexAttribI4uiv(GLuint index, VertexAttribUIVSource v);
```

Comment:
Should a similar change be made to vertexAttrib[1-4]fv family of functions in WebGL 1 specification?